core/spec/models/spree/log_entry_spec.rb

```diff
@@ -53,4 +53,16 @@
       expect { log_entry.parsed_details }.to raise_error(described_class::DisallowedClass, /log_entry_permitted_classes/)
     end
   end
+
+  describe '#parsed_details=' do
+    it 'serializes the provided value to YAML' do
+      value = double("an object", to_yaml: "---\nfoo: bar")
+
+      log_entry = described_class.new(parsed_details: value)
+
+      expect(value).to have_received(:to_yaml)
```

Comment:
Also non-blocking, but it's easy to decouple the test from the implementation by using any Ruby object as fixture (as `Object#to_yaml` is part of core). Thoughts?